documentation/general/dotnet-run-file.md

```diff
@@ -161,26 +172,37 @@ App/Program2/Program2.csproj
 The generated folders might need to be named differently to avoid clashes with existing folders.
 
 The entry-point projects (`Program1` and `Program2` in our example)
-have the shared `.cs` files source-included via `<Content Include="../Shared/**/*.cs" />`.
-We could consider having the projects directly in the top-level folder instead
-but that might result in clashes of build outputs that are not project-scoped, like `project.assets.json`.
-If we did that though, it would be enough to exclude the other entry points rather than including all the shared `.cs` files.
-
-Unless the [artifacts output layout][artifacts-output] is used (which is recommended),
-those implicit projects mean that build artifacts are placed under those implicit directories
-even though they don't exist on disk prior to build:
-```
-App/Program1/bin/
-App/Program1/obj/
-App/Program2/bin/
-App/Program2/obj/
+have the shared `.cs` files source-included via `<Compile Include="../Shared/**/*.cs" />`.
```

Comment:
Hm, this wouldn't include non-C# files like `.resx`. We could add items like `<EmbeddedResource Include="../Shared/**/*.cs" />` etc for all the default items, but I think I'm going to investigate if it would be possible to add a property like `<AdditionalDefaultIncludeItemsDirectory>..\Shared</>` that would be recognized by the .NET SDK props/targets.

Reply:
(I haven't done this in the corresponding implementation PR, I think we can implement this in a follow up.)